content-scripts/components/Accordion.tsx

```diff
@@ -0,0 +1,76 @@
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+import jsx from "texsaur";
+
+interface AccordionProps {
+    id: string;
+    title: string;
+    data: (string | JSX.Element)[];
+    max_size: number;
+}
+
+export const Accordion = ({
+    id,
+    title,
+    data,
+    max_size,
+}: AccordionProps): JSX.Element => {
+    const toggleAccordion = (event: MouseEvent) => {
+        const button: HTMLElement = event.currentTarget as HTMLElement;
+        const icon: HTMLElement = button.querySelector("i") as HTMLElement;
```

Comment:
This query should be more specific (using the id), otherwise we'll end up flipping all `i`